processor/transformprocessor/factory.go

```diff
@@ -48,6 +49,7 @@ func NewFactory() processor.Factory {
 
 func createDefaultConfig() component.Config {
 	return &Config{
+		ErrorMode:        ottl.PropagateError,
```

Comment:
I really want `Ignore` to be the default functionality.  Using it now would be a breaking change I think, but is that ok?  Do we need to follow a breaking change process for this?

Reply:
Ya we'll follow the process even if it means more work for me lol That means this PR is in the appropriate state.  In a future PR I can start the deprecation of `Propagate`.